03/ThorHoff/tags.py

```diff
@@ -0,0 +1,50 @@
+import xml.etree.ElementTree as ET
+from collections import Counter
+from difflib import SequenceMatcher
+
+TOP_NUMBER = 10
+RSS_FEED = 'rss.xml'
+SIMILAR = 0.87
+
+
+def get_tags():
+    """Find all tags in RSS_FEED.
+    Replace dash with whitespace."""
+    tag_list = []
+    tree = ET.parse(RSS_FEED)
+    root = tree.getroot()
+    for item in root.findall("channel/item/category"):
+        tag_list.append(item.text.replace("-", " ").lower())
+    return tag_list
+
+
+def get_top_tags(tags):
+    """Get the TOP_NUMBER of most common tags"""
+    tag_counter = Counter(tags)
+    return tag_counter.most_common(10)
```

Comment:
good use of counter but you used the magic number 10 instead of the global var, see docstring